script/audit_deps.py

```diff
@@ -22,6 +22,16 @@
     os.path.join('vendor', 'brave-extension', 'node_modules'),
 ]
 
+# Tag @sec-team before adding any advisory to this list
+# Ignore these rust advisories
+IGNORED_NPM_ADVISORIES = [
+# Remove when:
+# https://github.com/chronotope/chrono/issues/602 is resolved
+# Tracking issue: https://github.com/brave/brave-browser/issues/18838
+    'RUSTSEC-2020-0071',
```

Comment:
There is a fix available for this one (0.2.23): https://rustsec.org/advisories/RUSTSEC-2020-0071.html

Are we stuck on an earlier version?